src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List!</title> 
+</head>
+<body>
+    <header></header>
```

Comment:
Hi @HFG43 partner!, since this project didn't require any navbar or header of the like, I would suggest removing the `header` tag. However, I know that it might have been an idea to implement it later on, but for now and for the above reason as well as due to being empty, I kindly suggest removing it 👍🏼  

All in all your markup looks great! good job! 🎯